src/main/java/com/work/management/service/employee/impl/EmployeeServiceImpl.java

```diff
@@ -11,9 +11,12 @@
 import org.springframework.beans.BeanUtils;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Isolation;
+import org.springframework.transaction.annotation.Propagation;
+import org.springframework.transaction.annotation.Transactional;
 
 @Service
-final class EmployeeServiceImpl implements EmployeeService {
+public class EmployeeServiceImpl implements EmployeeService {
 
```

Comment:
Is removing final and making it public neccessary...try testing with all combination..like removing public...and removing public and making it final...just see..if it is working or not?

Reply:
Done